config.toml

```diff
@@ -3,8 +3,8 @@ address = ":8081"
 api_prefix = "/api/v1/"
 api_spec_file = "openapi.json"
 debug = true
-auth = false
-auth_type = "xrh"
+auth = true
+auth_type = "xhr"
```

Comment:
Why did you change it?

Reply:
I'll change it back. I used it for local testing

Reply:
Fixed